pkg/cmd/eventlistener/describe_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright © 2020 The Tekton Authors.
```

Comment:
There should be a test added for an `EventListener` with only a `Name`/`Namepsace`. Basically the minimum required fields specified.

Reply:
Added Testcase